namespaces/Representor.md

```diff
@@ -0,0 +1,44 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Element)
+
+The Resource represents a Hypermedia Resource, with its available transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `attributes` (object)
+    - `transitions` - (array[Transition]) - The available transitions from this resource.
+    - `resources` - (array[Resource]) - Any embedded (resources) from this resource.
```

Comment:
The reason why I think embedded resources go into the content is because embedded resources are actually part of the resource representation (though not part of the actual state server-side). This is actually probably a topic with lots of opinions, but if you embed a resource in the message, you are changing the message representation. A linked resource versus an embedded resource, though the same in theory, are actually different practically speaking.

It seems to me that you have two options that make sense:

1. Include it in content as said above with a special "embed" element
1. Have a "link" element that can contain the entirety of the linked resource, making it a link with some embedded data

With all of that said, I'm not against it being in attributes, though it is not really an attribute of the resource to me and (again) feels like an anti-pattern.